bindings/GeometryDash.bro

```diff
@@ -4623,7 +4639,27 @@ class MoreVideoOptionsLayer : FLAlertLayer {
 }
 
 [[link(android)]]
-class MultilineBitmapFont : cocos2d::CCSprite {}
+class MultilineBitmapFont : cocos2d::CCSprite {
+	static MultilineBitmapFont* create() = win 0x2a440;
+	static MultilineBitmapFont* createWithFont(const char* fontName, gd::string str, float scale, float width, cocos2d::CCPoint anchorPoint, int unk, bool bColourDisabled) = win 0x2a5b0;
+	gd::string readColorInfo(gd::string str) = win 0x2afb0;
+	void setOpacity(int) = win 0x2af50;
+	gd::string stringWithMaxWidth(gd::string str, float scaledWidth, float scale) = win 0x2b450;
+	bool initWithFont(const char* fontName, gd::string str, float scale, float width, cocos2d::CCPoint anchorPoint, int unk, bool bColourDisabled) = win 0x2a6a0;
+
+	float m_fontWidth[300];
```

Comment:
id say using std::array<float, 300> m_fontWidth; would be more preferable

Reply:
and probably a better name, since floatwidth implies a single value, not an array

Reply:
I think this is a mistake by the gd.h developers, there shouldn't be an array here.